Winapp2.ini

```diff
@@ -10176,11 +10191,18 @@ LangSecRef=3022
 Detect=HKLM\Software\Classes\JDownloader2
 DetectFile=%ProgramFiles%\JDownloader 2
 Default=False
-FileKey1=%ProgramFiles%\JDownloader*2*|JDownloader.log;Updater_*-*-*.log;SessionInstallLog.json.*|RECURSE
+FileKey1=%ProgramFiles%\JDownloader*2*|JDownloader.log;Updater_*-*-*.log;SessionInstallLog.json.*;output.log|RECURSE
 FileKey2=%ProgramFiles%\JDownloader*2*\captchas|*.*|RECURSE
 FileKey3=%ProgramFiles%\JDownloader*2*\cfg|CaptchaDialogDimensions_*.json
 FileKey4=%ProgramFiles%\JDownloader*2*\logs|*.*|RECURSE
 FileKey5=%ProgramFiles%\JDownloader*2*\tmp|*.*|RECURSE
+FileKey6=%ProgramFiles%\JDownloader*2*\.install4j|*.log
+FileKey7=%LocalAppData%\JDownloader*2*|JDownloader.log;Updater_*-*-*.log;SessionInstallLog.json.*;output.log|RECURSE
+FileKey8=%LocalAppData%\JDownloader*2*\captchas|*.*|RECURSE
+FileKey9=%LocalAppData%\JDownloader*2*\cfg|CaptchaDialogDimensions_*.json
+FileKey10=%LocalAppData%\JDownloader*2*\logs|*.*|RECURSE
+FileKey11=%LocalAppData%\JDownloader*2*\tmp|*.*|RECURSE
+FileKey12=%LocalAppData%\JDownloader*2*\.install4j|*.log
 ExcludeKey1=PATH|%ProgramFiles%\JDownloader*2*\cfg\|subconf_*.ejs

```

Comment:
Please always use Winapp2ool > WinappDebug to make sure the syntax and sorting is correct before opening a new pull request. Thank you very much.